File: doubleit/templates/doubleit.java
import java.io.*;

class Solution {
    /**
     * Return the total money Big Ben pays
     *
     * P: string of characters representing the people Big Ben talks to
     */
    static int solve(int L, String[] P) {
        // YOUR CODE HERE
        return -1;
    }
    
    static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);

    public static void main(String[] args) throws IOException {
        int T = Integer.parseInt(in.readLine());
        for (int i = 0; i < T; i++) {
            int L = Integer.parseInt(in.readLine());
            String[] P = in.readLine().split(" ");
            out.println(solve(L, P));
        }
        out.flush();
    }
}

File: doubleit/templates/doubleit.cpp
#include <iostream>
#include <vector>
#include <string>

using namespace std;

/**
 * Return the total money Big Ben pays
 *
 * L: vector of strings representing the people Big Ben talks to
 */
int solve(const vector<char>& L) {
    //YOUR CODE HERE
    return -1;
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        vector<char> L;
        string S;
        cin >> S;
        for (char c : S) {
            if (c != ' ') {
                L.push_back(c);
            }
        }
        cout << solve(L) << '\n';
    }
}
